src/ReflectionParameter.php

```diff
@@ -365,6 +374,23 @@ public function isVariadic()
     }
 
     /**
+     * Returns if default value set (or implied).
+     *
+     * Identical to isDefaultValueAvailable(), except it
+     * includes IMPLIED default values which:
+     *     + Only exist in builtin functions and methods.
+     *     + Only affect the optionality of a prameter, not
```

Comment:
Typo here in `prameter` word.

Reply:
👍 Spelling and typing will be the death of me... good catch.